pkg/materialize/connection_kafka.go

```diff
@@ -256,3 +216,47 @@ func (b *ConnectionKafkaBuilder) Create() error {
 
 	return b.ddl.exec(q.String())
 }
+
+// Brokers DDL
+func (b *ConnectionKafkaBuilder) BuildBrokersString() string {
```

Comment:
Abstracting the brokers DDL builder to a separate method, that way it can be used both in the Update/Alter and the Create funcs